package.json

```diff
@@ -60,8 +60,7 @@
     "winston": "^3.2.1",
     "yargs": "^15.4.1",
     "yasway": "^1.10.7",
-    "yuml2svg": "^4.2.1",
-    "z-schema": "^4.2.2"
```

Comment:
Those references to types in z-schema needs to be refactored too.

Reply:
This line is just remove extra "z-schema" dependency because "swagger-parser" and "yasway" both have "z-schema" dependency.

Security alert said it can't update "validator" to 13.7.0 because "validator" is under "z-schema", here's the list about "validator"
```
+-- swagger-parser@6.0.5
| `-- z-schema@3.25.1
|   `-- validator@10.11.0
`-- yasway@1.10.7
  `-- z-schema@4.2.3
    `-- validator@12.2.0
```
To update "validator" to 13.7.0, we can update "swagger-parser" to 10.0.3. But the latest version of "[yasway](https://www.npmjs.com/package/yasway)" is 1.10.7, so the alert will still be there.